docs/Nextor 3.0 Driver Development Guide.md

```
@@ -0,0 +1,1134 @@
+    ld a,<slot number or 0>
+    ex af,af'
+    xor a
+    ld ix,GWORK
+    call CALBNK
+```
+
+#### 4.2.6. CALDRV (4048h)
+
+This routine calls a routine in the driver bank (bank 7 for drivers embedded in ROM, that is, the value of [`K_SIZE`](#428-k_size-40feh)). It works like [`CALBNK`](#424-calbnk-4042h), except that the bank number is fixed and the address of the routine to call is taken from (BK4_ADD) instead of IX.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)